src/libraries/Microsoft.Extensions.Configuration.Binder/gen/ConfigurationBindingGenerator.Parser.cs

```diff
@@ -506,6 +506,42 @@ private TypeSpec CreateEnumerableSpec(TypeParseInfo typeParseInfo)
                 };
             }
 
+            private bool ConvertTo(ITypeSymbol source, ITypeSymbol dest)
+            {
+                var conversion = _typeSymbols.Compilation.ClassifyConversion(source, dest);
+                return conversion.IsReference && conversion.IsImplicit;
```

Comment:
Not super familiar with the `ClassifyConversion` APIs but am I right to guess that this just checks that `source` is a subtype of `dest`? If so, perhaps this could be named to something like "IsAssignableTo" since I would tend to to associate the term "conversion" with a broader spectrum of checks.

Reply:
I am fine renaming it to whatever. I was using the name we are using in the options source gen.

Reply:
Done renaming the method to `IsAssignableTo`